ibis/expr/tests/test_table.py

```diff
@@ -1049,6 +1049,22 @@ def test_union(table):
         t1.union(t3)
 
 
+def test_intersect(table):
+    schema1 = [('key', 'string'), ('value', 'double')]
+    schema2 = [('key', 'string'), ('key2', 'string'), ('value', 'double')]
+    t1 = ibis.table(schema1, 'foo')
+    t2 = ibis.table(schema1, 'bar')
+    t3 = ibis.table(schema2, 'baz')
```

Comment:
can you make a test_set_ops.py and move union tests here as well

Reply:
I would need to implement those in this pull request as well so should those be in this pull request? I was planning on adding those in a separate one but I can do it here

Reply:
ideally yes, but a followon pr is ok too